ServerCore/Pages/Submissions/FreeformQueue.cshtml.cs

```diff
@@ -91,7 +92,15 @@ orderby submission.TimeSubmitted
                 submissionQuery = submissionQuery.Where(submissionView => submissionView.PuzzleId == puzzleId);
             }
 
-            Submissions = await submissionQuery.ToListAsync();
+            Submissions = await submissionQuery.Take(50).ToListAsync();
```

Comment:
This does the 50 oldest, yes? And it does nice things when queue size is under 50, yes?

Would be nice to report the entire queue size, though. Maybe query the count when the take actually gets 50?

Reply:
Yes, the queue is oldest first (sorted above) and Take is a maximum. Added a query to get the full size.